src/app/components/flash/ErrorFlash.tsx

```diff
@@ -0,0 +1,39 @@
+import { AlertIcon, XIcon } from '@primer/octicons-react'
+import { Box, Flash, IconButton, Octicon } from '@primer/react'
+
+interface ErrorFlashProps {
+  message: string
+  closeFlash: () => void
```

Comment:
This can be an optional param to make this component both closable and non-closable and then we can get rid of `ErrorFlashNoClose`